### PR TITLE
text edits to holding, checkin  and projection

### PR DESCRIPTION
**Pull request by MrPyroMeiniac**

### CODE REVIEW

File: app/holding/index.html [hunk 1]
```diff
@@ -25,8 +25,8 @@
         <div class="card text-xs-center">
 
           <div class="card-block">
-            <h4 class="card-title">Please confirm</h4>
-            <p class="card-text geolocation-message"> <span class="extra-message"></span>Please tap the button below when the projector gives you the sign that we need your input</p>
+            <h4 class="card-title">ATTENTION!</h4>
+            <p class="card-text geolocation-message"> <span class="extra-message"></span>Tap the button below when the projector gives you the sign that we need your input</p>
```
Comment (kimberleynyam) [inline app/holding/index.html]: This all sounds really good, for line 29 how about "Tap the button below when the projector indicates that we need your input" or "signals that we need your input" - "gives you the sign that" sounds a little wordy

File: projections/index.html [hunk 2]
```diff
@@ -103,7 +103,7 @@
         <div>
 
           <h1>Warning</h1>
-          <p>You don't have long with david</p>
+          <p>You don't have long left with david</p>
```
Comment (kimberleynyam) [inline projections/index.html]: Probably sounding like a Year One teacher but is David without a capital intentional?

Comment (MrPyroMeiniac) [inline projections/index.html]: thanks Kim :smiley_cat:

File: app/checkin/checkin.js [hunk 3]
```diff
@@ -17,16 +17,15 @@ function processCheckin() {
...
+      // The lat and long of Southbank centre
+      var targetLat = 51.505533; 
+      var targetLon = -0.117014;
 
       // calculate the distance by passing our variables to our helper function getDistanceFromLatLonInKm(lat1, lon1, lat2, lon2) saved in helper-functions.js. The function returns a value in km that is the distance between two points. We assign this to a new variable, distance.
       var distance = getDistanceFromLatLonInKm(userLat, userLon, targetLat, targetLon);
 
       // Our logic. If the distance between our user and the target is less than 100m, show the login screen, else update the view with a helpful message
-      if (distance < 1000) {
+      if (distance < 100) {
```
Comment (aliblackwell) [inline app/checkin/checkin.js]: This is 100km still! 100metres would be 0.1 @MrPyroMeiniac

**Comment by aliblackwell** [comment]
@MrPyroMeiniac could we also look at https://github.com/bridgeatwaterloo/david/issues/105 in this PR? We need to tell the user to enable location services if they get a geolocation error.

Remember: `navigator.geolocation.getCurrentPosition(successCallback,` errorCallback, options)`

We are using an anonymous function for the successCallback:

```
navigator.geolocation.getCurrentPosition(function(position) {
  // success in here
});
```

So we'd need to do something like this:

```
navigator.geolocation.getCurrentPosition(function(position) {
  // success in here
}, function(error) {
  // error in here
});
```

or refactor it like this:

```

function gotLocation(position) {
  // success in here
}

function gotError(error) {
  // error in here 
}

navigator.geolocation.getCurrentPosition(gotLocation, gotError);
```

**Comment by aliblackwell** [comment]
Well done @MrPyroMeiniac @francissmedley !

**Pull request merged**